File: internal/write_files.py
folders = [
    {
        "path": "~/Library/Cache/Alice/messages",
        "contents": ["messages.db", "messages.db-shm", "messages.db-wal"],
    }
]

files_add = [
    {
        "path": "~/Library/Application Support/Alice/preferences/settings.json",
        "update": True,
    },
    {"path": "~/.config/zed", "update": True},
    {"path": "~/.gitconfig", "update": False},
    {"path": "~/.ssh", "update": False},
    {"path": "~/.XCompose", "update": False},
    {"path": "~/.zshrc", "update": True},
    {"path": "/etc/prime-discrete", "update": False},
]

files_remove = [
    "~/.bash_history",
    "~/.bash_logout",
    "~/.bashrc",
]

File: internal/write_files.go
package internal

// Folders is a slice of Folder structs representing folders and their contents.
var Folders = []Folder{
	{
		Path:     "~/Library/Cache/Alice/messages",
		Contents: []string{"messages.db", "messages.db-shm", "messages.db-wal"},
	},
}

// FilesAdd is a slice of FileAdd structs representing files to add and whether to update them.
var FilesAdd = []FileAdd{
	{
		Path:   "~/Library/Application Support/Alice/preferences/settings.json",
		Update: true,
	},
	{
		Path:   "~/.config/zed",
		Update: true,
	},
	{
		Path:   "~/.gitconfig",
		Update: false,
	},
	{
		Path:   "~/.ssh",
		Update: false,
	},
	{
		Path:   "~/.XCompose",
		Update: false,
	},
	{
		Path:   "~/.zshrc",
		Update: true,
	},
	{
		Path:   "/etc/prime-discrete",
		Update: false,
	},
}

// FilesRemove is a slice of strings representing files to remove.
var FilesRemove = []string{
	"~/.bash_history",
	"~/.bash_logout",
	"~/.bashrc",
}

// Folder represents a folder and its contents.
type Folder struct {
	Path     string
	Contents []string
}

// FileAdd represents a file to add and whether it should be updated.
type FileAdd struct {
	Path   string
	Update bool
}
